### PR TITLE
Fix Bug 1164572, add higher res hero image and SVG logo with fallback

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson @craigcook r?

### CODE REVIEW

File: bedrock/settings/static_media.py [hunk 1]
```diff
@@ -970,6 +970,13 @@
         ),
         'output_filename': 'js/contribute-form-bundle.js',
     },
+    'contribute-friends': {
+        'source_filenames': (
+            'js/libs/jquery-1.7.1.min.js',
```
Comment (craigcook) [inline bedrock/settings/static_media.py]: Does it specifically need to be this older version of jQuery? We've updated to 1.11.0 elsewhere, and I think 1.7 has some deprecated stuff that might throw errors.

File: bedrock/settings/static_media.py [hunk 2]
```diff
@@ -970,6 +970,13 @@
...
+            'js/mozorg/contribute/friends.js'
```
Comment (craigcook) [inline bedrock/settings/static_media.py]: I think this line needs a trailing comma

**Comment by craigcook** [comment]
Maybe we should also make a cropped down version of the hero image for small viewports, so it's a smaller file to download on mobile.

File: bedrock/settings/static_media.py (hunk 1 above)
Comment (schalkneethling) [inline bedrock/settings/static_media.py]: Oh crap no, I must have overlooked the other version. Will change.

**Comment by schalkneethling** [comment]
@craigcook yeah, @alexgibson also suggested that. Will do.

**Comment by alexgibson** [comment]
This looks good to me! Only nit is you can optimize the SVG to make it a bit smaller. I was able to shave off around 25% using https://jakearchibald.github.io/svgomg/

Other than that, r+wc from me! :cake:

**Comment by schalkneethling** [comment]
Optimized the SVG, just waiting for Travis. Thanks @alexgibson and @craigcook

**Pull request merged**